site-src/concepts/versioning.md

```diff
@@ -2,59 +2,87 @@
+point, these may graduate to the Standard release channel or be dropped from the
+API entirely.
+
+The following diagram provides a high level overview of the lifecycle of a
+feature or resource proposed by a new [GEP](/geps/overview/) (enhancement
+proposal) in Gateway API:
+
+```mermaid
+flowchart TD
+    0([Implementable GEP]) --> A
+    A>Experimental Channel] --> B([Widely used and working well?])
+    B -->|Yes| C>Standard Channel]
+    B -->|No| D([Could Changes Help?])
+    D -->|Yes| E([Adjust and try again])
+    D -->|No| F>Remove From API]
+    E -->A
+
+style A fill:#eeb
+style C fill:#beb
+style F fill:#ebb
+```
 
 The Standard release channel includes:
 
-* Resources that have graduated to beta
-* All fields that have graduated to standard and are no longer considered
-  experimental
+* Resources that have graduated to Beta or GA API Versions
+* All fields that have graduated to Standard from the Experimental channel
 
 The Experimental release channel includes everything in the Standard release
 channel, plus:
 
-* Alpha API resources
-* New fields that are considered experimental and have not yet graduated to the
-  standard channel
+* Resources with Alpha API Versions
+* All new fields before they graduate to Standard channel
 
 ![Release Channel Overlap](/images/release-channel-overlap.svg)
 <!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
 
-Release channels are used to indicate feature stability within Gateway API. For
-example, even if a resource has already graduated to beta, we need a way to test
-out new fields and features before committing to their long term stability. With
-upstream Kubernetes APIs, this is accomplished with [feature
-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
-Unfortunately, CRDs do not have a similar concept yet.
-
-Instead of trying to recreate feature gates in this project, we've introduced
-release channels. Starting in v0.5.0, all new fields and features will start in
-the Experimental release channel. From that point, these may graduate to the
-Standard release channel or be dropped from the API entirely. See
-[GEP-922](/geps/gep-922) for a more detailed discussion of this approach to new
-fields and features.
-
-![Lifecycle of New Gateway API Fields](/images/lifecycle-new-fields.svg)
-<!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
+We recommend using the "Standard" release channel by default as it will provide
+a stable experience. The "Experimental" feature enables us to iterate on new
+features quickly but may include breaking changes and should not be relied on
+in production environments.
+
+### API Versions
+Upstream Kubernetes APIs have 3 levels of stability, denoted by alpha, beta, and
+GA API versions. In Gateway API, we've narrowed this down to 2 levels of
+stability, expressed by our release channels as described above.
```

Comment:
Do you think it might be worth writing here the whys of this choice, along with the pros and cons?

Reply:
Added, thanks!